netbox-prometheus-sd.py

```diff
@@ -178,7 +178,7 @@ def discover_circuit(self):
 
 def main():
     format = "%(asctime)s %(filename)s [%(lineno)d][%(levelname)s] %(message)s"
-    logging.basicConfig(level=logging.DEBUG, format=format)
+    logging.basicConfig(level=logging.INFO, format=format)
```

Comment:
I find that hardcoding the logging in debug mode is a bad idea. I prefer to let it in "info" mode by default and to add an argument to enable the debug mode